kolibri/core/assets/src/vue/core-base.vue

```diff
@@ -99,7 +99,7 @@
         return this.navShown ? this.headerHeight * 4 : 0;
       },
       appBarStyle() {
-        return { paddingLeft: `${this.navWidth + PADDING}px` };
+        return this.mobile ? '' : { paddingLeft: `${this.navWidth + PADDING}px` };
```

Comment:
Feels more consistent to return an empty object rather than an empty string, but is the right fix.